src/EluvioLive.js

```diff
@@ -2266,17 +2265,6 @@ class EluvioLive {
       },
     ];
 
-    // Insert rarity if doesn't exist
```

Comment:
Is it useful to leave this code in here and just disable it?  Maybe make a constant 'insertRarity false' so we can turn on manually if needed in the future

Reply:
Latest commit has the insertRarity=false param and the code is left in there and only executes if true.